src/components/Tabs/TabsContext.tsx

```diff
@@ -1,3 +1,13 @@
 import * as React from 'react';
+import { TabStateReturn } from 'reakit/ts';
 
-export const TabsContext = React.createContext<any>({ selectedId: null });
+export interface TabsState {
+  tabState: TabStateReturn;
+  variant: 'default' | 'pill';
+}
+
+export const TabsContext = React.createContext<TabsState>({
+  // only will happen when tabs components are used outside of the context
+  tabState: {} as any,
```

Comment:
Nice, yeah thanks for cleaning this up.  I completely missed this when removing `selectedId` before 🙇